german/koeln/This is the story of koeln:

```diff
@@ -0,0 +1,4 @@
+This is the story of koeln:
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.